portal/common/lib/domain_parsing.ts

```diff
@@ -29,13 +29,18 @@ export function getSubdomainAndPath(url: URL): DomainDetails | null {
     {domain: name.walrus.site,
     { subdomain: "subname", path: "/index.html"}}
 */
-function splitUrl(url: URL): UrlExtract {
+function splitUrl(url: URL, portalNameLength?: Number): UrlExtract {
```

Comment:
Question: does this work when developing on localhost?

Reply:
Yes! Tests reassure us too! 👍